src/viam/ur/module/ur_arm.cpp

```diff
@@ -711,15 +712,25 @@ void URArm::move_joint_space_(std::shared_lock<std::shared_mutex> config_rlock,
     }
     segments.push_back(std::move(waypoints));
 
+    auto max_velocity = current_state_->get_speed();
+    if (options.max_vel_degs_per_sec && options.max_vel_degs_per_sec.get() > 0){
+        max_velocity = degrees_to_radians(options.max_vel_degs_per_sec.get());
+    }
```

Comment:
I need the `options.max_vel_degs_per_sec.get() > 0` because [rdk's server.go](https://github.com/viamrobotics/rdk/blob/2d2c71deb551f5c28470ffd636a554b57ff5de70/components/arm/server.go#L130) will always set the [velocity & acceleration to zero](https://github.com/viamrobotics/rdk/blob/2d2c71deb551f5c28470ffd636a554b57ff5de70/components/arm/pb_helpers.go#L14-L30). 

So every module receives MoveOptions with the velocity and acceleration set

Reply:
https://viam.atlassian.net/browse/RSDK-12375